qa/tasks/mgr/dashboard/helper.py

```diff
@@ -336,7 +336,17 @@ def _request(cls, url, method, data=None, params=None, version=DEFAULT_API_VERSI
             raise ex
 
     @classmethod
-    def _get(cls, url, params=None, version=DEFAULT_API_VERSION, set_cookies=False, headers=None):
+    def _get(cls, url, params=None, version=DEFAULT_API_VERSION, set_cookies=False, headers=None,
+             retries=0, wait_func=None):
+        while retries > 0:
+            try:
+                return cls._request(url, 'GET', params=params, version=version,
+                                    set_cookies=set_cookies, headers=headers)
+            except requests.ConnectionError:
+                log.info("Retrying... %s left", retries)
```

Comment:
```suggestion
                log.info("Retrying... %s left", retries)
                if retries == 0:
                    raise e from None  # to avoid the chained exception message
```